apps/namadillo/.env.sample

```diff
@@ -0,0 +1 @@
+COIN_GECKO_API_KEY=
```

Comment:
Agreed. This is only the simplest solution to enable the fiat feature. It's still not clear the definition of using the CoinGecko api, maybe we go with an alternative api. I'm planning to discuss this on the afternoon meeting

Reply:
I don't remember if we got to a final decision on this. But can we get rid of the .env file in order to let everything inside the config.toml? (i.e. when we redistribute a new Namadillo release, people can get the bundle and just change the config.toml instead of rebuild everything from scratch to add the CoinGecko or other API key)

Reply:
I'm not familiar with CoinGecko API, I'm supposing this is not a secret key :)